src/Service/WpInstallerService.php

```diff
@@ -18,6 +18,23 @@ protected function getInstalledVersion(): ?string
         return parent::getInstalledVersion() ?: $this->getLegacyInstalledVersion();
     }
 
+    /**
+     * Override because a null version will re-trigger a migration or overwrite all options with defaults
+     * when deactivating and re-activating the plugin in WordPress.
+     *
+     * @param  null|string $version
+     *
+     * @return void
+     */
+    protected function updateInstalledVersion(?string $version): void
+    {
+        if (! $version) {
```

Comment:
moet dit niet gewoon in de pdk kant al niet meer gebeuren? of desnoods op basis van config. vind het voor nu – vanwege haast – wel prima, maar dit is iets wat in andere platformen ook een issue zal zijn

Reply:
Ik twijfelde of dit bij andere platformen ook op die manier zou spelen. Maar is idd wel een aandachtspunt, scherp.